.gitignore

```diff
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# PyCharm, IntelliJ etc
+.idea/
+
+# Mac
+.DS_Store
```

Comment:
You opened the directory in finder?! Shame! 😏

Reply:
was testing some issues with file syncing le-sigh...